File: Java/Ropuszka.java
import java.util.Scanner;

public class Main 
{
    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();
        int n = input.nextInt();
        String result = (n == 33) ? "WRACAJ" : (a < b) ? "W LEWO" : "W PRAWO";
        System.out.print(result);
        input.close();
    }
}

File: C++/Ropuszka.cpp
#include <iostream>

int main() 
{
    int a, b, n;
    std::cin >> a >> b >> n;
    std::cout << (n == 33 ? "WRACAJ" : (a < b ? "W LEWO" : "W PRAWO"));
    return 0;
}